packages/devtools/src/ganache/server.ts

```diff
@@ -154,7 +154,11 @@ export class GanacheServer {
       }
     };
 
-    this.server = ganache.server({...serverOptions, _chainId: this.chainId} as any);
+    this.server = ganache.server({
+      ...serverOptions,
+      _chainId: this.chainId,
+      _chainIdRpc: this.chainId
```

Comment:
See https://github.com/trufflesuite/ganache-core/tree/master#options

Reply:
This might be good to specify in a comment

Reply:
30e8aef